server/ast/no_op.go

```diff
@@ -15,11 +15,24 @@
 package ast
 
 import (
+	"os"
+
+	"github.com/cockroachdb/errors"
 	vitess "github.com/dolthub/vitess/go/vt/sqlparser"
 
 	pgnodes "github.com/dolthub/doltgresql/server/node"
 )
 
+const ignoreUnsupportedEnvKey = "DOLTGRES_IGNORE_UNSUPPORTED"
+
+var ignoreUnsupportedStatements bool
```

Comment:
Should comment why we need this. I understand why, but can't have too much documentation for functionality. Someone else may not know why though in 2 years.